src/main/java/Task.java

```diff
@@ -0,0 +1,24 @@
+abstract class Task {
+    private String taskName;
+    private boolean isCompleted;
+
+
+    public Task (String taskName, boolean isCompleted) {
+        this.taskName = taskName;
+        this.isCompleted = isCompleted;
+    }
+
+    private String outputTaskStatus () {
+        return "[" + (isCompleted ? "X" : " ") + "] ";
+    }
+
+
+    @Override
+    public String toString () {
```

Comment:
Maybe could give a more detailed naming so it would understandable what toString actually does